lib/database_cleaner/active_record/truncation.rb

```diff
@@ -74,9 +74,10 @@ def has_been_used?(table)
           # Patch for MysqlAdapter with ActiveRecord 3.2.7 later
           # select_value("SELECT 1") #=> "1"
           select_value(<<-SQL).to_i > 1 # returns nil if not present
-          SELECT Auto_increment
+          SELECT auto_increment
           FROM information_schema.tables
-          WHERE table_name='#{table}';
+          WHERE table_name = '#{table}'
+          AND table_schema = database()
```

Comment:
@buehmann Good idea. Would you mind refactoring the SQL query into its own method?

Reply:
Sure, no problem.